qa/tasks/manypools.py

```diff
@@ -46,28 +46,28 @@ def task(ctx, config):
     poolprocs=dict()
     while (remaining_pools > 0):
         log.info('{n} pools remaining to create'.format(n=remaining_pools))
-	for remote, role_ in creator_remotes:
+        for remote, role_ in creator_remotes:
             poolnum = remaining_pools
             remaining_pools -= 1
             if remaining_pools < 0:
                 continue
             log.info('creating pool{num} on {role}'.format(num=poolnum, role=role_))
-	    proc = remote.run(
-	        args=[
-		    'ceph',
-		    '--name', role_,
-		    'osd', 'pool', 'create', 'pool{num}'.format(num=poolnum), '8',
-		    run.Raw('&&'),
-		    'rados',
-		    '--name', role_,
-		    '--pool', 'pool{num}'.format(num=poolnum),
-		    'bench', '0', 'write', '-t', '16', '--block-size', '1'
-		    ],
-		wait = False
-	    )
+            proc = remote.run(
+                args=[
+                    'ceph',
+                    '--name', role_,
+                    'osd', 'pool', 'create', 'pool{num}'.format(num=poolnum), '8',
+                    run.Raw('&&'),
+                    'rados',
+                    '--name', role_,
+                    '--pool', 'pool{num}'.format(num=poolnum),
+                    'bench', '0', 'write', '-t', '16', '--block-size', '1'
+                ],
+                wait = False
+            )
```

Comment:
As far as you touch each of the line, can this be replaced with human readable string, like?
```
proc = remote.run(args=
   "ceph --name {role} osd pool create pool{num} 8 && "
   "rados --name {role} --pool pool{num} bench 0 "
    "write -t 16 --block-size 1".format(role=role, num=poolnum), wait=False)
```

Reply:
I want to avoid that. My goal is to make flake8 happy for python2 and python3. I want to change as less as possible.